src/test/java/com/google/cloud/spanner/pgadapter/StatementParserTest.java

```diff
@@ -54,7 +54,7 @@ public void testRemoveCommentsAndTrim() {
             + "  INSERT VALUES (v.column1, v.column2, v.column3)\n"
             + "WHEN MATCHED\n"
             + "  UPDATE SET FirstName = v.column2,\n"
-            + "             LastName = v.column3";
+            + "             LastName = v.column3;";
```

Comment:
Similar comment to above, are we sure this won't cause a problem?

Reply:
Sending the ';' with the query doesn't impact it.  I'm not sure why the comment stripping code removed it in the first place.